packages/lit-dev-content/site/docs/libraries/bundles.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Bundles
+eleventyNavigation:
+  key: Bundles
+  parent: Related libraries
+  order: 4
+---
+
+Lit is available as pre-built, single-file bundles. These bundles are useful for
+situations where you can't use npm or just want to experiment without setting up
+a build process.
+
+<div class="alert alert-warning">
+
+**If you're using npm for client-side dependencies, you should use the [`lit`
+package](https://www.npmjs.com/package/lit) instead of these bundles.** The
+bundles intentionally combine most or all of Lit into a single file. This can
+prevent some build tools from removing the parts of Lit your project doesn't
+use, which means your users may download more code than they strictly need.
+
+</div>
+
+## Download
+
+The bundles can be found on GitHub in [the lit/dist
+repository](https://github.com/lit/dist). To download a particular version, go
+to [the repository's tags](https://github.com/lit/dist/tags), find the version
+you want, and click either the "zip" or "tar.gz" link to download an archive of
+the bundles in the chosen format.
+
+## Usage
+
+The archive has multiple folders, each containing a bundle and its source map.
+The bundles are standard JavaScript modules with no dependencies - any modern
+browser should be able to import and run the bundles directly.
+
+For example, if you extracted the bundles into a directory called `lit-bundles`
+next to one of your source files, you could use the `core` bundle in that source
+file like this:
+
+```js
+import {LitElement, html} from './lit-bundles/core/lit-core.min.js';
+```
+
+### Source maps
```

Comment:
I don't think we need the detailed info here. Just that the bundles include source maps is enough info (e.g. you must also not edit this file, but we don't say that).

Reply:
I think we can remove mention of this all together. Our primary published sources have source maps and we don't document it there.